### DISCUSSION TITLE
twitter只能浏览视频和文字无法加载图片，v2rayNG1.9.6

### DISCUSSION THREAD

**Discussion by Floreal0** [2dust/v2rayNG#3788] [Q&A]
v2rayNG1.9.6版本，打不开谷歌商店，推特软件上打不开图片，edge经常提示连接不上网络。手机是vivo

**Comment by FF-crazy** [comment]
这是你的机场的问题，不是v2r的问题

**Reply by Floreal0** [reply]
但是当我使用在线订阅转换工具将机场转到clash上使用的时候就没有任何问题
